gel-cli-instance/src/cloud/instance.rs

```diff
@@ -23,11 +24,21 @@ struct CloudInstanceBackup<H: CloudHttp> {
 }
 
 impl<H: CloudHttp> InstanceBackup for CloudInstanceBackup<H> {
-    fn backup(&self, callback: ProgressCallback) -> Operation<Option<BackupId>> {
+    fn backup(
+        &self,
+        strategy: RequestedBackupStrategy,
+        callback: ProgressCallback,
+    ) -> Operation<Option<BackupId>> {
         let api = self.instance.api.clone();
         let name = self.instance.name.clone();
 
         tokio::spawn(async move {
+            if strategy != RequestedBackupStrategy::Auto {
```

Comment:
Hm, manual cloud backups should be supported, is that not the case?

Reply:
It should be supported -- in this case the strategy is full/incremental/auto, and I figured cloud was "auto" on. BackupType is where we get manual/automatic.

Auto for strategy just means "pick incremental if we have a full backup and it's not too far away".

Hard to name the enums, but type was already in place for manual/auto :)